### ISSUE TITLE
Request: The blue hand

### ISSUE THREAD

**Issue by Coupe70** [ideoforms/AbletonOSC#110]
When you select a device in Live it gets the blue hand symbol and the first up to 16 parameters of the device 
will be automapped to a MIDI controller if there is a remote script running for this controller. But the data to 
query is not exactly 'selected device', because you can lock the blue hand to a device, decoupling it from the
selected device.

The data to query seems to be a song property called 'appointed device'.
I would assume that it returns track number and device number.
I don't think there are dedicated parameters like 'blue hand device parameter 1', instead you would use the
already implemeted device calls to handle the blue hand device.

So this would "only" mean to implement get/set(?)/listen appointed device.

**Comment by ideoforms** [comment]
Yep, looks like this would be a Song property, `appointed_device` (from [docs](appointed_device)). Would need a bit of additional logic to allow the API user to specify the object path of the device (e.g., track 2 device 3) and vice versa.

**Comment by hems** [comment]
And you can also right click and lock different controllers to different devices!

It would be great if we could do that sort of assignment dynamically!

For instance lock control surface 1 to a certain rack and then later lock to a different tack in a different track!
